app/utils/__tests__/checkAccess.test.ts

```diff
@@ -0,0 +1,30 @@
+import checkAccess from '../checkAccess';
+
+describe('checkAccess', () => {
+  const mockUserGroups = [
+    {
+      name: 'clinician',
+      ident: 'uuid1',
+    },
+    {
+      name: 'manager',
+      ident: 'uuid2',
+    },
+  ];
+
+  test('Allow access when both allow and block list is empty', () => {
```

Comment:
does this mean a user can view data without having roles at all?

Reply:
If the develop does not give an explicit `blockList`, but rather just undefined/null, then `checkAccess` defaults to blocking all. However, if the user provides a `[]` as the input for `blockList`, then it's open to everything